.changeset/breezy-moles-reply.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core': patch
+'@backstage/plugin-auth-backend': patch
```

Comment:
This'll be a breaking change for some, assuming the `default` scope is something that some implementations actually depend on. Best make it a `minor` change and include instructions for how to migrate